.storybook/main.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+	stories: ['../client/**/*.stories.@(tsx|jsx)'],
+	addons: ['@storybook/addon-essentials', '@storybook/addon-viewport', '@storybook/addon-knobs'],
+};
```

Comment:
**Important change** — Storybook v6 uses a main.js file to configure story paths and addons.